frontend/src/components/header/Header.js

```diff
@@ -1,56 +1,59 @@
-import React, {Component} from 'react';
-import {Grid} from '@material-ui/core';
-import logo from '../../assets/ethBuenosAires.png';
+import React, { Component } from 'react'
+import { Grid } from '@material-ui/core'
+import logo from '../../assets/ethBuenosAires.png'
+import logoLight from '../../assets/ethBuenosAires-light.svg'
 import { Link } from 'react-router-dom'
 
 import './header.scss'
 
-
-
 export default class HeaderWhite extends Component {
-    render() {
-        let className = this.props.white ? 'main-header-white' : 'main-header'
-        return (
-            <div className={className}>
-                <Grid container>
-                    <Grid item xs={3}>
-                        <img src={logo} className={"logo"} alt={"logo"}/>
-                    </Grid>
-                    <Grid item xs={6}>
-                        <Grid container>
-                            <Grid item xs={3}>
-                                <Link
-                                    to="/"
-                                    rel="noopener noreferrer"
-                                    className="text"
-                                >HOME</Link>
-                            </Grid>
-                            <Grid item xs={3}>
-                                <Link
-                                    to="/events"
-                                    rel="noopener noreferrer"
-                                    className="text"
-                                >EVENTS</Link>
-                            </Grid>
-                            <Grid item xs={3}>
-                                <Link
-                                    to="/education"
-                                    rel="noopener noreferrer"
-                                    className="text"
-                                >EDUCATION</Link>
-                            </Grid>
-                            <Grid item xs={3}>
-                                <Link
-                                    to="/ecosystem"
-                                    rel="noopener noreferrer"
-                                    className="text"
-                                >ECOSYSTEM</Link>
-                            </Grid>
-                        </Grid>
-                    </Grid>
-                    <Grid item xs={3}/>
-                </Grid>
-            </div>
-            );
-    }
-}
\ No newline at end of file
+  render() {
+    const { white: lightTheme, selected } = this.props
+    return (
+      <div className={this.props.white ? 'main-header-white' : 'main-header'}>
+        <Grid container>
+          <Grid item xs={3} className="header-logo">
+            <img src={lightTheme ? logoLight : logo} className={'logo'} alt={'logo'} />
```

Comment:
Let's improve the `alt` attribute to improve accessibility changing it to the logo name I guess will be `Ethereum Buenos Aires`